tests/nemo_text_processing/en/data_inverse_text_normalization/test_cases_money_cased.txt

```diff
@@ -0,0 +1,52 @@
+Two dollars~$2
+One cent~$0.01
+Four united states dollars and sixty nine cents~$4.69
+Seventy five dollars sixty three~$75.63
+Twenty nine dollars fifty cents~$29.50
+Eleven dollars and fifty one cents~$11.51
+Nine hundred ninety three dollars and ninety two cents~$993.92
+Four hundred sixty billion won~₩460 billion
+Thirty billion yen~¥30 billion
+Two point five billion dollars~$2.5 billion
+Forty five billion dollars~$45 billion
+Fifty million dollars~$50 million
+Fifty billion dollars~$50 billion
+Zero point two million dollars~$0.2 million
```

Comment:
Can you add zero point two Million 
Dollars


One Cent

Reply:
added